extensions/azure-functions-http/maven-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -80,15 +80,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <artifactId>maven-surefire-plugin</artifactId>
-        <version>${surefire-plugin.version}</version>
-        <configuration>
-          <systemProperties>
-            <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
-          </systemProperties>
-        </configuration>
-      </plugin>
```

Comment:
This doesn't look right? I don't see any parent here.

Reply:
Oops, fixed.